RocketDataTests/DataModelManagerTests.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  DataModelManagerTests.swift
+//  RocketData
+//
+//  Created by Peter Livesey on 10/20/16.
+//  Copyright © 2016 LinkedIn. All rights reserved.
+//
+
+import XCTest
+import RocketData
+
+class DataModelManagerTests: RocketDataTestCase {
```

Comment:
Please add a test that would fail to compile without this change (so if/when swift fixes the bug we can go back to generics and know it works). The test case would be a good location to document what the bug is and how/why you worked around it.